send-data/cloudwatch.mdx

```diff
@@ -28,12 +29,88 @@ Axiom CloudWatch Forwarder includes templates for the following CloudFormation s
 {/* list separator */}
 - [Create an account on AWS Cloud](https://signin.aws.amazon.com/signup?request_type=register).
 
-## Setup
+## Installation
+
+To install the Axiom CloudWatch Forwarder, choose one of the following:
+
+- [Cloudformation stacks](#install-with-cloudformation-stacks)
+- [Terraform module](#install-with-terraform-module)
+
+### Install with Cloudformation stacks
 
 1. [Launch the Forwarder stack template on AWS](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=axiom-cloudwatch-forwarder&templateURL=https://axiom-cloudformation.s3.amazonaws.com/stacks/axiom-cloudwatch-forwarder-v1.1.1-cloudformation-stack.yaml). Copy the Forwarder Lambda ARN because it’s referenced in the Subscriber stack.
 1. [Launch the Subscriber stack template on AWS](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=axiom-cloudwatch-subscriber&templateURL=https://axiom-cloudformation.s3.amazonaws.com/stacks/axiom-cloudwatch-subscriber-v1.1.1-cloudformation-stack.yaml).
 1. [Launch the Listener stack template on AWS](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=axiom-cloudwatch-listener&templateURL=https://axiom-cloudformation.s3.amazonaws.com/stacks/axiom-cloudwatch-listener-v1.1.1-cloudformation-stack.yaml).
 
+### Install with Terraform module
+
+<Steps>
+<Step>
+
+Create a new module for the Forwarder in your Terraform file in the following way:
+
+```hcl
+module "forwarder" {
+  source           = "axiomhq/axiom-cloudwatch-forwarder/aws//modules/forwarder"
+  axiom_dataset    = "DATASET_NAME"
+  axiom_token      = "API_TOKEN"
+  prefix           = "axiom-cloudwatch-forwarder"
+}
+```
+
+<Replacement />
+Alternatively, use Axiom’s provider to create a dataset.
+
+</Step>
+<Step>
+
+Create a new module for the Subscriber in your Terraform file in the following way:
+
+``` hcl
+module "subscriber" {
+  source               = "axiomhq/axiom-cloudwatch-forwarder/aws//modules/subscriber"
+  prefix               = "axiom-cloudwatch-forwarder"
+  forwarder_lambda_arn = module.forwarder.lambda_arn
+  log_groups_prefix    = "/aws/lambda/"
+}
+```
+
+</Step>
+<Step>
+
+Create a new module for the Listener in your Terraform file in the following way:
+
+```hcl
+module "listener" {
+  source               = "axiomhq/axiom-cloudwatch-forwarder/aws//modules/listener"
+  prefix               = "axiom-cloudwatch-forwarder"
+  forwarder_lambda_arn = module.forwarder.lambda_arn
+  log_groups_prefix    = "/aws/lambda/"
+}
+```
+
+</Step>
+<Step>
+In your terminal, go to the folder of your main Terraform file, and then run `terraform init`.
+</Step>
+<Step>
+Run `terraform plan` to check the changes, and then run `terraform apply`.
+</Step>
+<Step>
+To remove subscriptions from the Forwarder, use the Unsubscriber module.
```

Comment:
How to use this? Just create a new module in the Terraform file? Why can't users create this earlier?

Reply:
So, this shouldn't be part of initial setup as this removes the subscriptions from the Forwarder, meaning, it disabled the log forwarding for the selected log groups. Users might not need to do that at all, and if they do it won't be during setup.